spec/fixtures/federal_register_documents.yml

```diff
@@ -147,3 +147,16 @@
   publication_date: 2013-02-01
   comments_close_on: <%= Date.current.advance(days: 2).to_s(:db) %>
   significant: false
+
+'2022-12345':
+  federal_register_agencies: fr_irs
+  document_number: 2022-12345
+  title: Fishing for Refunds
```

Comment:
Let's tweak the title as some of the other fixtures do, to call out what this spec is testing, i.e.:
```
  title: Fishing for Refunds, irrelevant due to significance/old pub date/insignificant/doc type
  ```
  ...or whatever seems clear and fits in the `title` field